### DISCUSSION TITLE
Output units in SOLWEIG

### DISCUSSION THREAD

**Discussion by hosseinamini1994** [UMEP-dev/UMEP#541] [Q&A]
Hi All,

I recently computed the Mean Radiat Temperature using SOLWEIG, and the output values for Tmrt ranged between 36 and 77. I was presenting the outcomes to my supervisor, but I realized there is nothing on UMEP's website that confirms if the unit of these values is in Fahrenheit or Celcius. I am guessing it's in Fahrenheit (because it doesn't make much sense if it's in Celcius) but I was wondering how I can confirm it.

Thank you for your time and help in advance!

**Comment by nilswallenberg** [comment]
Hi,

Mean radiant temperature (Tmrt) is in Celsius. You can read more about Tmrt here:

https://link.springer.com/article/10.1007/s00484-008-0162-7

https://rmets.onlinelibrary.wiley.com/doi/10.1002/joc.1537

Cheers,
Nils
